### PR TITLE
Add plugin: Daily ICalendar

### PR DESCRIPTION
**Pull request by mmmykhailo**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/mmmykhailo/obsidian-daily-icalendar

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

File: community-plugins.json [hunk 1]
```diff
@@ -9098,5 +9098,12 @@
      "author": "Mgussekloo",
      "description": "Autofocus the editor when opening a new file.",
      "repo": "mgussekloo/obsidian-editor-autofocus"
+  },
+  {
+     "id": "daily-icalendar-plugin",
```
Comment (mmmykhailo) [inline community-plugins.json:9103]:
```suggestion
     "id": "daily-icalendar",
```

**Comment by mmmykhailo** [comment]
Bruh, github failed uploading them for some reason 🤨

**Comment by joethei** [comment]
[import { VCalendar, VEvent, VTimeZone } from "node-ical";](https://github.com/mmmykhailo/obsidian-daily-icalendar/blob/be02da932ba8f8c9dd82eac469d3fe3bb4369840/utils/html.ts#L1)
This import is only available from a Node.js runtime, which means that this will throw errors on Mobile (that's a problem since you have `isDesktopOnly` marked as `false` in your manifest.json file.

**Comment by mmmykhailo** [comment]
I set isDesktopOnly to true for now. Don't have enogh time to find a better library or write my own.
By the way, if you know one that have good enough types - I would be glad if you advised it 🤓

**Pull request merged**